src/attached_probe.cpp

```diff
@@ -191,7 +191,7 @@ AttachedProbe::AttachedProbe(Probe &probe,
       // If BPF_PROG_TYPE_RAW_TRACEPOINT is available, no need to attach prog
       // to anything -- we will simply BPF_PROG_RUN it
       if (!feature.has_raw_tp_special())
-        attach_uprobe(safe_mode);
+        attach_uprobe(0, safe_mode);
```

Comment:
I'm not sure that this is correct. IIUC, we use process' PID in this case, set in [bpftrace.cpp:1090](https://github.com/iovisor/bpftrace/blob/master/src/bpftrace.cpp#L1090). This should probably use the same approach.